domain/src/main/java/eu/europa/ec/fisheries/uvms/bean/AssetDomainModelBean.java

```diff
@@ -253,6 +253,15 @@ public Asset getAssetHistory(AssetHistoryId historyId) throws AssetModelExceptio
         return EntityToModelMapper.toAssetFromAssetHistory(assetHistory);
     }
 
+    public List<Asset> getAssetHistories(List<String> guids) throws AssetModelException {
+        if (guids == null || guids.isEmpty()) {

```

Comment:
Why don't you return an empty list here? Just asking, I don't mind leaving it like this.